setup.py

```diff
@@ -259,9 +260,117 @@ def run(self):
         remove_sprites_from_folder(self.buildfolder / "data" / "sprites" / "alttpr")
 
         self.create_manifest()
-
         if is_windows:
             # Inno setup stuff
+            # did not find a good source to automatically pull these
+            default_rom_file_names: typing.Dict[str, str] = {
```

Comment:
Should we ever get "AutoWorldSettings", this could pull from there. Until then `Utils.get_default_options()`, however we still need a dict "A Link to the Past" -> "lttp_options" here (which is cleaner than putting the rom name here imo)